introduce new dataset directory (#22)
C0C0AN/musicgenre_similarities: dataset/readme.md

# NC2U dataset

> Within here, you'll find resources related to the dataset aspects of the project.

Reviewer: ```suggestion
This directory provides resources related to the dataset aspects of the project.
```

Reviewer: Looks good, just a minor suggestion. We're talking about a directory, right?
